source/documentation/onboarding.html.md

```diff
@@ -1,26 +1,144 @@
+  and how you will participate.
+
+<a name="join-irc-channel"/>
+
+### Join IRC channel #rdo on Freenode server
+* All the development and communication related to RDO Project happens on #rdo IRC channel on
+  Freenode server. Click [this link](http://webchat.freenode.net/?channels=#rdo) to join the same and feel free to say Hello!, we are always there to help.
+* Feel free to check out the [IRC etiquette](https://www.rdoproject.org/community/irc-etiquette/)
+
+ <a name="github-account"/>
+
+### Create Github account
+ Go to [Github Sign Up](https://github.com/join) and enter your username, email address and
+ password and you are done. If you already have a GitHub account, you can skip this step.
+
+<a name="sign-in"/>
+
+### Sign to review.rdoproject.org
+Sign-in to [review.rdoproject.org](https://review.rdoproject.org/auth/login) using github account.
+
+<a name="add-ssh"/>
+
+### Add SSH key to review.rdoproject.org
+* Generate your SSH key, if you don't have.
+  Follow these instructions on [Generating a new SSH key](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/#generating-a-new-ssh-key)
+* Then, go to [ssh-key](https://review.rdoproject.org/r/#/settings/ssh-keys) and click on "Add Key" button and copy the contents of id_rsa.pub (SSH Key public key) and paste it there and then click on 'Add' button and you are done.
+
+<a name="bugzilla-account"/>
+
+### It would be good to have Red Hat Bugzilla account.
+All the bugs related to RDO packages are tracked on Red Hat Bugzilla under *RDO* product. OpenStack projects bugs are tracked on [Launchpad](https://launchpad.net/openstack).
+* You can create a Bugzilla account by clicking on [Create a new Red Hat Bugzilla account](https://bugzilla.redhat.com/createaccount.cgi)
+  and enter your email id and click on "send" button and check your mailbox and you are done.
+
+<a name="git-review"/>
+
+### Configure your git and git-review.
+* Run these steps to let git know about your email address:
+ ```
+$ git config --global user.the name "Firstname Lastname"
+$ git config --global user.email "your_email@youremail.com"
+ ```
+* To check your git configuration:
+```
+$ git config --list
+```
+* Install git-review tool
+```
+$ sudo yum install git-review
+```
+
+<a name="rdopkg"/>
+
+### Install rdopkg tool
+Follow this link to install [rdopkg](https://www.rdoproject.org/documentation/intro-packaging/#rdopkg)
+And you are set for making the contribution to RDO Project.
+
+<a name="what-next"/>
+
+## **So what next**
+* RDO Project has lots of easy fixes. Check out the [RDO easyfix](https://github.com/redhat-openstack/easyfix/issues)
+  and pick one issue and feel free to move ahead.
+* Clone the project.
+For cloning DLRN project, you need DLRN source code URL
+```
+$ git clone https://github.com/softwarefactory-project/DLRN.git
+```
+You can also use rdopkg to clone an RDO package.
+```
+$ rdopkg clone -u <github user name> package-name
```

Comment:
Option "-u <username>" should not be needed if git is configured properly as done in previous section.